viewer/src/main/webapp/viewer-html/common/viewercontroller/ol/OlVectorLayer.js

```diff
@@ -636,6 +708,17 @@ Ext.define("viewer.viewercontroller.ol.OlVectorLayer", {
             style.featureStyle = featureStyle.config;
             olFeature.setStyle(style);
         }
+    },
+
+    keyDown: function (event) {
+        if (event.key === "s" || event.key === "S") {
+            this.drawRightAngle = true;
+        }
+    },
+
+    keyUp: function (event) {
+        if (event.key === "s" || event.key === "S") {
+            this.drawRightAngle = false;
+        }
```

Comment:
is er een reden om voor "S" te kiezen?... of "gewoon" (niet dat ik een bepaalde voorkeur heb voor een andere key)

Reply:
Nee geen reden voor, andere knop is prima als dat nodig is.